sdk/communication/Azure.Communication.Chat/src/ChatThreadClient.cs

```diff
@@ -138,12 +138,22 @@ public virtual Response<ChatThreadProperties> GetProperties(CancellationToken ca
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
         public virtual async Task<Response<SendChatMessageResult>> SendMessageAsync(string content, ChatMessageType type = default, string senderDisplayName = null, CancellationToken cancellationToken = default)
+        {
+            return await SendMessageAsync(new SendChatMessageOptions { Content = content, MessageType = type, SenderDisplayName = senderDisplayName }, cancellationToken).ConfigureAwait(false);
+        }
+
+        /// <summary> Sends a message to a thread asynchronously. </summary>
+        /// <param name="options"> Options for the message. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
+        public virtual async Task<Response<SendChatMessageResult>> SendMessageAsync(SendChatMessageOptions options, CancellationToken cancellationToken = default)
```

Comment:
May I know what the reason is to combine content and properties in a new model `SendChatMessageOptions`  instead of just adding a pass-in parameter `properties` on `SendMessage` and `UpdateMessage` method?